### PR TITLE
Added secondary header html

### PR DESCRIPTION
**Pull request by ruthvik947**
Secondary header (Banana/Facetview navbar) html file exists now. JQuery script to generate is also in the file (commented out). Feel free to (i.e. please do) mess around with the CSS to get proper alignment/a better look. This is the best solution I could find without having to modify our current nav-bar.

@titusjung @janethkim

### CODE REVIEW

File: html/secondary-header.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<div id="nav-secondary">
+  <nav class="navbar nav-main navbar-default" style="background-color: #000;">
+    <ul class="nav navbar-nav navbar-custom nav-ul-second">
+      <li><a class="navlink" href="http://polar.usc.edu/Polar/banana/#/dashboard">Banana</a></li>
+      <li><a class="navlink" href="http://polar.usc.edu/Polar/facetview/">FacetView</a></li>
+    </ul>
+  </nav>
+</div>
+
+<!-- Use this script to generate secondary header -->
```
Comment (chrismattmann) [inline html/secondary-header.html]: if commented out, remove it

Comment (ruthvik947) [inline html/secondary-header.html]: @chrismattmann, it's a script that Janet/Titus can use to render the header wherever they need to. Thought it'd be useful so I put it in there. (not used there - just a sort of guide. Can move to anywhere if you want though)

Comment (chrismattmann) [inline html/secondary-header.html]: I would just add it to docs, or something. But shouldn't be part of this PR.

**Comment by lorsposto** [comment]
hey @ruthvik947 is this still in prog?

**Comment by chrismattmann** [comment]
looks like my feedback has been addressed thanks @ruthvik947 @lorsposto
